src/components/layout/Layout.jsx

```diff
@@ -32,7 +32,9 @@ class Layout extends Component {
     return (
       <IntlProvider locale="en">
         <>
-          <Helmet titleTemplate="%s - edX" defaultTitle="edX" />
+          <Helmet titleTemplate="%s - edX" defaultTitle="edX">
+            <html lang="en" />
```

Comment:
is this something that would need to be configurable if/when we get to i18n?

Reply:
Yeah, I was looking at how prospectus does their i18n. They use a plugin called `gatsby-plugin-intl` to configure their messages, default locale, etc. But since i18n hasn't been a requirement for us (yet), just doing the minimal amount of work needed to resolve that a11y error of missing "lang" attribute.